javascript-modal/main.js

```diff
@@ -0,0 +1,16 @@
+var $openBtn = document.querySelector('.modalBtn');
+var $modal = document.querySelector('.modal');
+var $container = document.querySelector('.container');
+
+$openBtn.addEventListener('click', function event() {
+  $container.classList.toggle('active');
+  $modal.style.display = 'flex';
+  $openBtn.classList.toggle('inactive');
+});
+
+var $nobtn = document.querySelector('.nobtn');
+$nobtn.addEventListener('click', function e() {
```

Comment:
You are naming the function `e` here, but I assume you meant that as a parameter.